---
layout: page-fullwidth
title: "Research"
teaser: My research focuses on understanding the biogeochemical functioning and physiological ecology of eukaryotic phytoplankton in a changing ocean. In particular, I am interested in the maintenance of genetic diversity in these populations and the role of biodiversity in ecosystem functioning. I use a combination of culture- and field-based experiments and 'omic techniques to address these questions.
permalink: "/research/"
header:
   image_fullwidth: "headers/Core.jpg"
---
<br>
<br>
  <h3>Identifying differential resource utilization between two closely related species</h3>
<div class="row">
  <div class="medium-8 columns">
The vast diversity of the phytoplankton has long perplexed biological oceanographers, as these organisms superficially appear to coexist in an isotropic environment while competing for the same basic resources: nutrients and light. Partitioning of resources has been hypothesized to be one factor enabling the “paradox of the plankton,” but quantitative approaches to identify and track it in the field have been lacking. Sequencing approaches such as metatranscriptome profiling enable species-specific metabolic profiles to be tracked within a complex, mixed assemblage. Using this approach enabled the direct examination of the ecophysiology of specific species <i>in situ</i>. Additionally, the development of a novel metatranscriptome pipeline enabled further comparisons between species by using these data to 1) identify novel resource responsive gene targets without a priori knowledge of function and 2) contextualize expression signals to compare the ecophysiology of organisms. 
</div>
  <div class="medium-4 columns">
  <img src="https://github.com/halexand/halexand.github.io/blob/master/images/CellMap_NB.png?raw=true" alt="">
</div>
</div>

<br>
<br>

 <h3>Characterizing physiological constraints on the oligotrophic rare biosphere</h3>
<div class="row">
  <div class="medium-4 columns">
  <img src="https://github.com/halexand/halexand.github.io/blob/master/images/diatom_heat.png?raw=true" alt="">
  </div>
  <div class="medium-8 columns">
Low nutrient, oligotrophic subtropical gyres cover such vast regions of the planet that they are central to global carbon cycling. In these low productivity systems, injection of nutrient rich water has been found to stimulate the typically dilute eukaryotic phytoplankton, specifically diatoms, that then form large, productive blooms. These diatom blooms cause dramatic increases in primary production and carbon export to the deep sea. Despite their critical importance, little is known about what causes these blooms or the metabolic traits that favor blooms of diatoms over other functional groups of phytoplankton. By tracking <i>in situ</i> expression patterns over time in the oligotrophic North Pacific Subtropical Gyre and comparing them to simulated blooms I am working to identify the metabolic basis of functional group specific traits that drive the eukaryotic phytoplankton bloom dynamics in the open ocean.
</div>
</div>

<br>
<br>
 <h3>Physiological ecology of a bloom-forming <i>Coscinodiscus</i> species in the field and in culture</h3>
<div class="row">
  <div class="medium-8 columns">
  During the <a href="http://deep-dom.blogspot.com/">DeepDOM</a> cruise, we transited through the Amazon plume. The Amazon river is responsible for ~15% of all fresh water input to the world's oceans. Consequently, this region represents a unique biogeochemical environment, characterized by low salinity and rich in terrestrial carbon and nutrients. In the low salinity lens of the plume, we discovered a blooming species of <i>Coscinodiscus</i>, a centric diatom. We were able to monitor its physiological status <i>in situ</i>, perform field incubation experiments, and recover and isolate from this bloom to study in culture.
    </div>
  <div class="medium-4 columns">

    <img src="https://github.com/halexand/halexand.github.io/blob/master/images/Cosci.png?raw=true" alt="">
   </div>
</div>

<h3> Future work: Quantitative unification of multi-omic datasets</h3>

The ‘omic revolution has shifted the limiting step in understanding the microbial role in biogeochemical cycles away from difficulty in directly monitoring the metabolism of microbial communities to data integration. We must be able to quantitatively unify both physical and chemical environmental data with large ‘omic’ datasets of all varieties (from genomic to metabolomic) if we wish to address larger questions of ecology. This is a computational and quantitative problem that is of great interest to me as I believe it will allow us not only to better address overarching questions, but that cross use of these data will inform the analysis and interpretation of each individual data type. 

